stdlib/random.ml

```diff
@@ -40,6 +40,35 @@ module State = struct
     let s = create () in
     set s i1 i2 i3 i4; s
 
+  let serialization_prefix =
+    (* '1' is a version number; more than 200 versions may fit a char. *)
```

Comment:
I didn't get that comment. Does that mean that the version is a byte and that the smallest version is 0x49 ?

Reply:
I don't have a real plan, but we have the option to change this character to indicate incompatible formats of the LXM algorithm, and that there is enough room for many such versions if we want to. (I think we should wait to see the Random algorithm evolve to decide what kind of prefixing/versioning strategy we actually want -- possibly none. In the meantime I like having a self-descriptive format and keeping some wiggle room in it.)